locales/zh_CN/LC_MESSAGES/hello.po

```diff
@@ -457,7 +438,7 @@ msgstr ""
 
 #: ../../hello.rst:213
 msgid "``break`` **is not supported in parallel loops**:"
-msgstr "**并行** 循环不支持 ``break`` 语句："
+msgstr "**并行循环不支持** ``break`` 语句："
```

Comment:
Is this right?

Reply:
This is the original translation ..... 😂

Reply:
Looks like I forgot to save the document after the last check :(